charts/apiclarity/templates/deployment.yaml

```diff
@@ -76,11 +76,13 @@ spec:
               value: {{ .Values.apiclarity.notificationURLPrefix }}
             {{- end }}
             - name: FUZZER_DEPLOYMENT_TYPE
-              value: kubernetes
+              value: {{ .Values.apiclarity.env.plugins.FUZZER_DEPLOYMENT_TYPE }}
```

Comment:
is it not already being placed during 
```
          {{- range $key, $val := .Values.apiclarity.env.plugins }}
            - name: {{ $key }}
              value: {{ $val | quote }}
          {{- end }}
```
?

Reply:
you're right